src/scenes/Languages/Create/CreateLangaugeForm.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { CreateLanguageInput } from '../../../api';
+import {
+  DialogForm,
+  DialogFormProps,
+} from '../../../components/Dialog/DialogForm';
+import { SubmitError, TextField } from '../../../components/form';
+
+export type CreateLanguageFormProps = DialogFormProps<CreateLanguageInput>;
+
+export const CreateLanguageForm = (props: CreateLanguageFormProps) => (
+  <DialogForm
+    DialogProps={{
+      fullWidth: true,
+      maxWidth: 'xs',
+    }}
+    {...props}
+    title="Create Language"
+  >
+    <SubmitError />
+    <TextField
+      name="language.name"
+      label="Name"
+      placeholder="Enter language"
+      autoFocus
+      required
+    />
+    <TextField
+      name="language.displayName"
+      label="Display Name"
+      placeholder="Enter Display Name"
+      autoFocus
```

Comment:
We don't want to autofocus every field - only one. Typically the first one.

Reply:
Changed